src/components/navigation/section-document-or-browser.tsx

```diff
@@ -136,32 +134,44 @@ export const SectionDocumentOrBrowser: React.FC<IProps> = observer(function Sect
     );
   };
 
-  const showPlayback = user.type ? appConfigStore.enableHistoryRoles.includes(user.type) : false;
+  //TODO: Need to refactor this if we want to deploy to all tabs
   const renderDocumentView = (subTab: ISubTabSpec) => {
     const openDocumentKey = tabState?.openDocuments.get(subTab.label) || "";
     const openDocument = store.documents.getDocument(openDocumentKey) ||
       store.networkDocuments.getDocument(openDocumentKey);
+    const publishedDoc = openDocument?.type === "publication" || openDocument?.type === "personalPublication"
+                          || openDocument?.type === "learningLogPublication";
+    const showPlayback = user.type && !publishedDoc ? appConfigStore.enableHistoryRoles.includes(user.type) : false;
+    const isStarredTab = selectedSubTab.label === "Starred";
 
-    if (!openDocument || openDocument.getProperty("isDeleted")) return false;
-
+    if (!isStarredTab && (!openDocument || openDocument.getProperty("isDeleted"))) return false;
```

Comment:
```suggestion
    const skipDocument = !openDocument || openDocument.getProperty("isDeleted");
    if (!isStarredTab && skipDocument) return false;
```
Then use `skipDocument` on line 156 as well.
I also wanted to double check again that the logic on line 147 is correct. As written, we'll continue on here if the document is starred, even if it's not open or it's deleted.